### ISSUE TITLE
error with wan lora after conversion for use with comfyui

### ISSUE THREAD

**Issue by selectai255** [kohya-ss/musubi-tuner#207]
hi i need some help 
ive created a couple of lora's for hunyuan but having trouble with creating wan lora's

after conversion i get this error on all blocks and the lora isn't loaded . this is using native comfy nodes 

ERROR lora diffusion_model.blocks.16.self_attn.q.weight shape '[5120, 5120]' is invalid for input of size 2359296
ERROR lora diffusion_model.blocks.16.self_attn.k.weight shape '[5120, 5120]' is invalid for input of size 2359296
ERROR lora diffusion_model.blocks.16.self_attn.v.weight shape '[5120, 5120]' is invalid for input of size 2359296
ERROR lora diffusion_model.blocks.16.self_attn.o.weight shape '[5120, 5120]' is invalid for input of size 2359296
ERROR lora diffusion_model.blocks.16.cross_attn.q.weight shape '[5120, 5120]' is invalid for input of size 2359296
ERROR lora diffusion_model.blocks.16.cross_attn.k.weight shape '[5120, 5120]' is invalid for input of size 2359296
ERROR lora diffusion_model.blocks.16.cross_attn.v.weight shape '[5120, 5120]' is invalid for input of size 2359296
ERROR lora diffusion_model.blocks.16.cross_attn.o.weight shape '[5120, 5120]' is invalid for input of size 2359296
ERROR lora diffusion_model.blocks.16.ffn.0.weight shape '[13824, 5120]' is invalid for input of size 13762560
ERROR lora diffusion_model.blocks.16.ffn.2.weight shape '[5120, 13824]' is invalid for input of size 13762560

training comand is 

accelerate launch --num_cpu_threads_per_process 1 --mixed_precision fp16 wan_train_network.py --task t2v-1.3B --dit "L:\New folder\ComfyUI\models\diffusion_models\wan\wan2.1_t2v_1.3B_fp16.safetensors" --dataset_config wan.toml --sdpa --mixed_precision fp16 --fp8_base --optimizer_type adamw8bit --learning_rate 1e-4 --fp8_t5 --fp8_base --gradient_checkpointing --max_data_loader_n_workers 2 --persistent_data_loader_workers --network_module networks.lora_wan --network_dim 32 --network_alpha 32 --timestep_sampling shift --discrete_flow_shift 3.0 --max_train_steps 2000 --save_every_n_steps 100 --seed 42 --network_args loraplus_lr_ratio=4 --logging_dir=logs --output_dir "L:\musubi\musubi-tuner\output" --output_name wan6

hunyuan lora's were 230mb in size trained with 26 images but wan are coming out at only 85mb ( with same dataset ) and after conversion were only 1kb until i changed the model to fp16 .

been trying on and off for a few weeks if anyone can help it would be appreciated

**Comment by Sarania** [comment]
Do you have the latest Musubi? I'd start there. Otherwise I note you're training the 1.3B model which I haven't done personally(or seen done much) and it may be related. The size of your LoRA is a function of it's network dimension and the size of the base model. It's normal that a dim 32 for Hunyuan (~13 billion parameters) would be larger than one for Wan 1.3B (~1.3 billion). Coming out as 1kb is not normal, and the fact that changing the model to fp16 helped may suggest it's due to some recent issues that popped up when implementing support for fp16 models in the first place. If updating doesn't help, hopefully kohya has an idea.

**Comment by selectai255** [comment]
hi thanks for the reply , yes i fully updated musubi and deleted old environment and started a fresh one , tried to install all requirements but pillow 10.2 wouldnt install and had to get latest don't know if that matters ,  forgot to say i'm on windows 10 if thats relevant .

**Comment by selectai255** [comment]
hi thanks for reply , this is comand i use for conversion 

python convert_lora.py --input output/wan6.safetensors --output outputconv/wan6.safetensors --target other

**Comment by selectai255** [comment]
hi just wanted to add that this is not strictly correct "and after conversion were only 1kb until i changed the model to fp16" .
initially using bf16 they were converting to the correct size but didn't work then i must have changed something and they began converting to 1kb then when i changed to fp16 they started converting to the correct size again but still didn't work. 

like i said i've been trying for a couple of weeks and not sure what i've changed on different try's  .

i have tried musubi with the gui where i created sample's during training and the lora was working so i think it is definitely the conversion where its failing .
thanks again for any help.

**Comment by selectai255** [comment]
i will do some testing in that respect and report back

**Comment by selectai255** [comment]
done a bit of testing today , converting a converted lora (which stayed at 85mb) back to default it again stayed at 85mb and using generate.py it had an effect on the resulting video .

loaded the same lora into kijai image to vid w/flow with 1.3b model also had an effect on the genarated video. 

so the lora's are being created ok it's just when converting to use with natiive comfyui that errors occur 

btw i read something kijai wrote yesterday about the diif synth lora's not needing to be converted for native comfyui because comfyui now supports them. does this mean lora's created with musubi no longer need converting or are diff synth a different format to begin with . haven't had chance to update comfy and test a non converted lora yet .
